OpenRA.Game/Player.cs

```diff
@@ -210,21 +210,24 @@ public Color PlayerStanceColor(Actor a)
 			}
 
 			return stanceColors.Neutrals;
-		}
```

Comment:
Ugh must be line endings. I ran dos2unix on it and it removed more new line characters :(